README.md

```
@@ -1,12 +1,13 @@
 # PictoPy
 
-PictoPy is an advanced desktop gallery application that combines the power of Tauri, React, and Rust for the frontend with a Python backend for sophisticated image analysis and management.
+PictoPy is an advanced desktop gallery application that combines the power of Tauri, React, and Rust on the frontend with a Python backend for sophisticated image analysis and management.
 
-# Want to Contribute? 😄
+
+## Want to Contribute? 😄
 
 &nbsp;&nbsp;&nbsp;<a href="https://discord.gg/hjUhu33uAn"><img src="https://github.com/user-attachments/assets/3ed93273-5055-4532-a524-87a337a4fbba" height="40"></a>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)